### PR TITLE
Updated the latest version in README.

### PR DESCRIPTION
**Pull request by anandology**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,23 +5,27 @@ Visit http://webpy.org/ for more information.
 [![build status](https://secure.travis-ci.org/webpy/webpy.png?branch=master)](https://travis-ci.org/webpy/webpy)
 [![Codecov Test Coverage](https://codecov.io/gh/webpy/webpy/branch/master/graphs/badge.svg?style=flat)](https://codecov.io/gh/webpy/webpy)
 
-The latest stable release `0.51` supports both Python 2.7 and Python >= 3.5.
+The latest stable release `0.60` only supports supports Python >= 3.5.
```
Comment (iredmail) [inline README.md:8]: Duplicate “supports”?

File: README.md [hunk 2]
```diff
@@ -5,23 +5,27 @@ Visit http://webpy.org/ for more information.
...
+```
+# For Python 3
+python3 -m pip install web.py==0.60
+```
+
+If you are using Python 2, please use the 0.51 version. Please note that
+this is the last version that supports Python 2. 
 ```
 # For Python 2.7
 pip2 install web.py==0.51
```
Comment (cclauss) [inline README.md:20]:
```suggestion
python3 -m install web.py<0.60
```
This should work even if we have to release another release for legacy Python.

File: README.md (hunk 1 above)
Comment (anandology) [reply README.md:8]: Thanks for pointing that. I'll correct it.

@iredmail Do you mind, if I merge this and fix it in the subsequent commit?

Comment (anandology) [reply README.md:8]: I want to tag this commit and 0.60, as I've already published the release.

File: README.md (hunk 2 above)
Comment (iredmail) [reply README.md:20]: I didn’t use “python2 -m install” before ...

File: README.md (hunk 1 above)
Comment (iredmail) [reply README.md:8]: Sure no problem.

File: README.md (hunk 2 above)
Comment (anandology) [reply README.md:20]: I guess he meant `python2 -m pip install`

File: README.md (hunk 1 above)
Comment (anandology) [reply README.md:8]: could you please aprove this PR?

File: README.md (hunk 2 above)
Comment (anandology) [reply README.md:20]: I think it is better to mention the explicit verson. I'm going to change this to:

```
python2 -m pip install web.py==0.51
```

**Pull request merged**